UpdateCgnOrchestrator/index.ts

```diff
@@ -1,236 +1,9 @@
-﻿import { IOrchestrationFunctionContext } from "durable-functions/lib/src/classes";
+﻿import * as df from "durable-functions";
+import { getConfigOrThrow } from "../utils/config";
+import { handler } from "./handler";
```

Comment:
can we use a more detailed name for this functions, es: updateCgnOrchestratorHandler